packages/api/src/utility.ts

```diff
@@ -220,6 +220,7 @@ export function countRichtextChars(blocksCharLength: number, nodes: any) {
 }
 
 export const MAX_COMMENT_LENGTH = 1000
+export const MAX_PAYLOAD_SIZE = '1MB'
```

Comment:
what is the default? Isn't 1MB very low? And about what kind of payload are we talking here?

Reply:
@michael-scheurer since its the text size its not to low ;-) Or how many text documents you have that are over 1M ;-)